src/Command/GenerateObjectAclCommand.php

```diff
@@ -128,7 +134,8 @@ public function execute(InputInterface $input, OutputInterface $output)
             }
 
             $manipulatorId = sprintf('sonata.admin.manipulator.acl.object.%s', $admin->getManagerType());
-            if ($manipulator = $this->aclObjectManipulators[$manipulatorId] ?? null) {
+            $manipulator = $this->aclObjectManipulators[$manipulatorId] ?? null;
```

Comment:
```suggestion
            if (!$manipulator = $this->aclObjectManipulators[$manipulatorId] ?? null) {
```

Reply:
Assign and validation is not a good idea